content/en/installing-pixie/03-install-guides/01-community-cloud-for-pixie.md

```diff
@@ -5,15 +5,19 @@ metaDescription: "Getting started guide to setup Pixie"
 order: 1
 ---
 
-Please review Pixie's [requirements](/installing-pixie/requirements) to make sure that your Kubernetes cluster is supported. 
+## Prerequisites
+
+- Review Pixie's [requirements](/installing-pixie/requirements) to make sure that your Kubernetes cluster is supported.
+
+- Determine if you already have an [operator lifecycle manager](https://docs.openshift.com/container-platform/4.5/operators/understanding/olm/olm-understanding-olm.html) (OLM) deployed to your cluster, possibly to the default `olm` namespace. Pixie uses the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to manage its Vizier, which handles data collection and query execution (see the [Architecture](/about-pixie/what-is-pixie/#system-architecture) diagram). The OLM is used to install, update and manage the Vizier Operator.
```

Comment:
I would make this capitalized: "Operator Lifecycle Manager" and remove "an", so it's clear that if there were some other "operator manager" that someone writes, we specifically need OLM.